aasemble/django/apps/api/tests.py

```diff
@@ -306,3 +305,43 @@ def test_github_push_event(self, poll_one):
         from .tasks import github_push_event
         github_push_event("https://github.com/eric/project0")
         poll_one.delay.assert_called_with(1)
+
+class APIv2TagsTests(APIv1Tests):
+    list_url = '/api/v2/snapshots/'
+
+    def test_create_snapshot_tag(self):
+        data = {'mirrorset': 'http://testserver/api/v2/mirror_sets/60d0ba66-d343-404b-a6e6-5c141db11a54/',
+                'tags':[{'tag':'non-default'}]}
```

Comment:
This feels awkward.

If there are multiple tags, would I provide something like: `'tags': [{'tag': 'firsttag'}, {'tag': 'secondtag'}]`?

It really should just be `'tags': ['firsttag', 'secondtag']`.